src/PIL/ImageOps.py

```diff
@@ -248,7 +248,8 @@ def contain(image, size, method=Image.Resampling.BICUBIC):
     :param size: The requested output size in pixels, given as a
                  (width, height) tuple.
     :param method: Resampling method to use. Default is
-                   :py:attr:`PIL.Image.BICUBIC`. See :ref:`concept-filters`.
+                   :py:attr:`PIL.Image.Resampling.BICUBIC`.
```

Comment:
This renders like `PIL.Image.Resampling.BICUBIC`, which is quite long:

<img width="716" alt="image" src="https://user-images.githubusercontent.com/1324225/211147184-332dda91-33d1-486d-b9b6-ca3959aa565c.png">

We _could_ do:

```suggestion
                   :py:attr:`~PIL.Image.Resampling.BICUBIC`.
```
Which renders as `BICUBIC`:

<img width="711" alt="image" src="https://user-images.githubusercontent.com/1324225/211147211-e22e094b-b98b-475a-819e-3d137bf49640.png">

I know that chops off the Resampling, which is sort of the point here, but it's linking to the proper one, so I think it's fine?

I don't have a strong preference, feel free to ignore these suggestions :)

Reply:
Thanks. I was only changing ImageOps because after the other changes, the reference target for ":py:attr:\`PIL.Image.BICUBIC`" was no longer found.